docs/contributing.md

```diff
@@ -51,7 +55,7 @@ To contribute to the frontend, first install the JavaScript dependencies. The mo
 
 ```bash
 cd packages/framework
-yarn install
+yarn
```

Comment:
I think it's better to recommend using the full command, even though yarn on its own does the same thing.

```suggestion
yarn install
```